mst/page-index/page-index.js

```diff
@@ -11,6 +11,10 @@ module.exports = [
         title: "About Managed Service for TimescaleDB",
         href: "about-mst",
       },
+      {
```

Comment:
The second page is missing from the page-index :)

Reply:
added the child pages